### PR TITLE
 ✨ Add image resize cloud function

### PR DESCRIPTION
**Pull request by williamchong007**

### CODE REVIEW

File: src/util/api/index.js [hunk 1]
```diff
@@ -28,6 +28,12 @@ export const getCivicLikerTrialEventByIdAPI = id =>
 export const getCivicLikerJoinTrialEventByIdAPI = id =>
   `/api/civic/trial/events/${id}/join`;
 
+export const getImageResizeAPI = (url, { width } = {}) =>
+  `https://us-central1-civic-liker${
+    IS_TESTNET ? '-develop' : ''
+  }.cloudfunctions.net/thumbnail/?url=${encodeURIComponent(url)}${
```
Comment (nwingt) [inline src/util/api/index.js]: Should pass host in env var?

Comment (williamchong007) [reply src/util/api/index.js]: probably want to use it in dev env too to avoid unwanted diff in behaviour...
not sure

Comment (nwingt) [reply src/util/api/index.js]: can others use that API publicly? If so, is that a concern?

File: packages/resizeImageHttp/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  IS_TESTNET,
+} = process.env;
+
+const app = express();
+
+app.use(helmet());
+app.get('/', async (req, res) => {
+  const { url } = req.query;
+  const { origin } = req.headers;
+  if (!IS_TESTNET && origin && !origin.includes(ORIGIN_DOMAIN)) {
```
Comment (nnkken) [inline packages/resizeImageHttp/index.js]: What if the origin header is not in the request?
Should use CORS here to enforce?

Comment (williamchong007) [reply packages/resizeImageHttp/index.js]: browser does not enforce cors on img src

Comment (nnkken) [reply packages/resizeImageHttp/index.js]: So the checking is only for best effort?

File: src/util/api/index.js (hunk 1 above)
Comment (williamchong007) [reply src/util/api/index.js]: probably can as in we didnt put any real checking here except origin header, and this url is very visible under browser network inspector

File: packages/resizeImageHttp/index.js (hunk 2 above)
Comment (williamchong007) [reply packages/resizeImageHttp/index.js]: yes unless we check against db record of og image

**Pull request merged**